src/memory/AmdFlash.cc

```diff
@@ -659,7 +724,8 @@ void AmdFlash::serialize(Archive& ar, unsigned version)
 	ar.serialize("ram",    *ram,
 	             "cmd",    cmd,
 	             "cmdIdx", cmdIdx,
-	             "state",  state);
+	             "state",  state,
+	             "status", status);
```

Comment:
And this should also be handled with bw-compat code.
Older savestates won't have `status` yet.

Search for `versionAtLeast` for examples of how to do this.

Reply:
Ah, I didn’t think too much of it, thought it would be tolerant of added / removed properties like I believe Unreal Engine’s serialisation is. Will change.